src/DesignBoard/wwwroot/css/styles.css

```diff
@@ -1,7 +1,9 @@
 .old {
-    background-color: orange;
+    background-color: #f09609
```

Comment:
This orange still clashes a little with the panel tinting provided by bootstrap. Not sure what the best way to handle that is. Maybe the panel tinting is enough?